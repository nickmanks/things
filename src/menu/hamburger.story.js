/* global module */
import React from 'react';
import {storiesOf} from '@storybook/react';
import Hamburger from './hamburger';
import 'hamburgers/dist/hamburgers.min.css';


storiesOf('Simple/Hamburger', module)
  .add('inactive hamburger', ()=> (
    <Hamburger isActive={false} />
  ))
  .add('active hamburger', ()=> (
    <Hamburger isActive={true} />
  ));
