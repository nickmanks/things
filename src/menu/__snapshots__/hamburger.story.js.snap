// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Simple/Hamburger active hamburger 1`] = `
<button className={"hamburger hamburger--squeeze is-active"} type={"button"}>
  <span className={"hamburger-box"}>
    <span className={"hamburger-inner"} />
  </span>
</button>
`;

exports[`Storyshots Simple/Hamburger inactive hamburger 1`] = `
<button className={"hamburger hamburger--squeeze"} type={"button"}>
  <span className={"hamburger-box"}>
    <span className={"hamburger-inner"} />
  </span>
</button>
`;
